implementations/rust/ockam/ockam_app/src/invitations/commands.rs

```diff
@@ -83,6 +83,18 @@ pub async fn create_service_invitation<R: Runtime>(
     .await
     .map_err(|e| e.to_string())?;
 
+    // send the invitation asynchronously to avoid blocking the application waiting for a result
+    let app_clone = app.clone();
+    tauri::async_runtime::spawn(async move { send_invitation(invite_args, app_clone).await });
+    app.trigger_global(super::events::REFRESH_INVITATIONS, None);
```

Comment:
Shouldn't this trigger be inside the async closure?

Reply:
yes indeed, done.